docs/guide/third-party.md

```diff
@@ -3,11 +3,13 @@
 ## Vuetify
 
 ### Installation
+
 Before we start please install vuetify and the webpack css-loader by running `npm install vuetify --save --save-exact && npm install --save-dev --save-exact css-loader` and follow the [Clean-Up-Guide](/guide/clean-up.md).
 
 ### Configure Webpack
 
-`./config/webpack.base.config.js`:
+`./config/base.js`:
```

Comment:
how about renaming `config` folder to `webpack`? more explicit.

Reply:
I like it 👍